src/gwt/src/org/rstudio/studio/client/workbench/views/packages/ui/InstallPackageDialog.java

```diff
@@ -73,6 +74,8 @@ public InstallPackageDialog(
       server_ = server;
       globalDisplay_ = globalDisplay;
 
+      RStudioThemes.disableDarkMenus();
```

Comment:
For symmetry, it seems like this should occur in the `onLoad()` call rather than the constructor.